### ISSUE TITLE
Custom UI draw commands cause spaming in the Output log after a compile

### ISSUE THREAD

**Issue by Evildea** [FlaxEngine/FlaxEngine#546]
So, I'm using drawing commands in my own UI Control. Everytime I compile this causes the output log to spam errors. I need to press play then stop to stop this from happening.

You can see it in the video here:
https://user-images.githubusercontent.com/35228400/120571294-24014e00-c45d-11eb-8dae-50a8e44633c8.mp4

**Flax version:**
Master Branch

**Comment by jb-perrier** [comment]
You should check out what's your code at line 34, looks like it's where it throws the null obj exception.

**Comment by Evildea** [comment]
I can't see any issue with my code. This also only happens after a compile.

The following is my code:
```
        public FontReference Font;
        public string Dialogue;
        public Vector2 Position;
        public Margin Margin;
        public Color BubbleBackgroundColor;
        public Color BubbleFontColor;
        public int MaxLength = 25;

        /// <inheritdoc />
        public override void Draw()
        {
            base.Draw();

            //var features = Render2D.Features;
            //Render2D.Features = Render2D.RenderingFeatures.None;

            // Don't draw anything if there is no Dialogue.
            if (Dialogue == "" || BubbleBackgroundColor == null || BubbleFontColor == null || Font.Font == null)
                return;

            // Get the Font Size.
            string Temp = WrapString(Dialogue.Trim(), MaxLength);

            // Draw the Bubble
            Vector2 FontSize = Font.GetFont().MeasureText(Temp);
            Vector2 MainPivot = DrawBubbleCircle(FontSize, CountLines(Temp));

            // Draw the Dialogue Text.
            Render2D.DrawText(Font.GetFont(), Temp, BubbleFontColor, MainPivot + new Vector2(0f, 18f));

            //Render2D.Features = features;
        }

        string WrapString(string stringToSplit, int maximumLineLength)
        {
            return Regex.Replace(stringToSplit, @"(.{1," + maximumLineLength + @"})(?:\s|$)", "$1\n");
        }
```

**Comment by mafiesto4** [comment]
Fixed in 79b03f6b5cb9c747e0faff634be552644cb6cfee.
